mapviz_plugins/src/laserscan_plugin.cpp

```diff
@@ -324,6 +324,31 @@ namespace mapviz_plugins
     point_size_ = static_cast<size_t>(value);
   }
 
+  void LaserScanPlugin::updatePreComputedTriginometic(const sensor_msgs::LaserScanConstPtr& msg)
+  {
+      static size_t prev_size      = 0;
+      static float  prev_angle_min = msg->angle_min;
```

Comment:
Please avoid the use of local static variables by refactoring these to class variables or creating a new object to encapsulate the entire trigonometry cache.

Reply:
I understand this is "personal taste". Or is there any technical reason to avoid static?
No problem, though

Reply:
I understand you point now. You are right